crypto/dalek/simd/Cargo.toml

```diff
@@ -0,0 +1,15 @@
+[package]
+name = "mc-crypto-dalek-backend-simd"
+description = "MobileCoin Dalek Crypto Backend Helper (SIMD)"
+version = "2.0.0"
+authors = ["MobileCoin"]
+edition = "2021"
+
+
+[features]
+default = [ "curve25519-dalek" ]
+
+[dependencies]
+curve25519-dalek = { version = "4.0.0-pre.2", optional=true, default-features = false, features = ["nightly", "simd_backend"] }
+ed25519-dalek = { version = "2.0.0-pre.1", optional=true, default-features = false, features = ["nightly", "simd_backend"]  }
+x25519-dalek = { version = "2.0.0-pre.2", optional=true, default-features = false, features = ["nightly", "u64_backend"] }
```

Comment:
`u64_backend` looks out of place here. Intended?

Reply:
goood question, x25519-dalek doesn't expose a `simd_backend` [feature](https://github.com/dalek-cryptography/x25519-dalek/blob/main/Cargo.toml#L58), so i copied this from the prior target detection